src/index.d.ts

```diff
@@ -0,0 +1,25 @@
+// index.d.ts
+declare module "@zerodevx/svelte-toast";
+
+import type { SvelteComponent } from "svelte";
+
+export interface SvelteToastOptions {
```

Comment:
Do we add hints for default property values here?

Reply:
Ah I see, if it's not too much trouble, could you add the JSDOC parts? (or perhaps the scaffolding and I can fill in the blanks - that will help a lot too.) Thanks for this!

Reply:
HI @Abourass, just a ping to see if you could scaffold the JSDOC parts into the typescript definition file too?

Reply:
@zerodevx - Sorry about that, slipped my mind. Got that finished and added